gamemodes/terrortown/gamemode/client/cl_vskin/vgui/dnumslider_ttt2.lua

```diff
@@ -195,7 +202,33 @@ function PANEL:SetConVar(cvar)
 
 	self.conVar = GetConVar(cvar)
 
-	self:SetValue(self.conVar:GetFloat())
+	self:SetValue(self.conVar:GetFloat(), true)
+end
+
+---
+-- @param string cvar
+-- @realm client
+function PANEL:SetServerConVar(cvar)
+	if not cvar or cvar == "" then return end
+
+	self.serverConVar = cvar
+
+	cvars.ServerConVarGetValue(cvar, function (wasSuccess, value)
```

Comment:
Yeah thats a point. I didnt want to hardcode the necessary convars and instead make it on demand only.
I will look into ttt2net, but generally I thought of something like a buffer, that gets checked every frame and and the end send all requests at once.
But it would only do this once per F1 menu, as they get cached. So gotta see, whats the easiest approach 😄

Reply:
Okay, the buffered requests should solve the problem.
Now only needed variables are sent to clients, that really need those values. Should minimize traffic I hope.